.github/workflows/exercise-tests.yml

```diff
@@ -10,7 +10,7 @@ on:
 
```

Comment:
Re the line endings, if I run the "check for windows line endings" command I found on Stackoverflow for this branch, it still shows a file with the wrong line endings:
```
find . -not -type d -exec file "{}" ";" | grep CRLF
=> ./exercises/practice/forth/.meta/tests.toml: ASCII text, with CRLF line terminators
```
Maybe see how it goes after that one is fixed as well.

Reply:
Could you again check if that is still the case? If I run `find . -not -type d -exec file "{}" ";" | grep CRLF` I don't get any files

Reply:
It's all good now.

For the general issue, I found this post the most "insightful" so far: https://github.com/golangci/golangci-lint/issues/1018#issuecomment-618594466

The gofmt linting complains about 3 files "at random" on Windows, not 3 specific ones. The default in golangci-lint is to only report 3 errors.
I think there is something that goes wrong when the git checkout is done on windows by the CI. Maybe it is possible to find out what setting for autocrlf is applied by the github action and whether we can change it to something else.

Reply:
Using `git config --system core.autocrlf false` seems to really fix it, can we keep this? Thanks for providing the link to the issue.

Reply:
Yes, I think that is a good solution. Well done! If we would prescribe such a setting to every contributor there might be issues but since it is only for the CI windows maschine, I think its totally fine.